src/Helpers/Tap/TappedTabBar.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  ActionType,
+  ContextModule,
+  OwnerType,
+  TappedTabBar,
+} from "../../Schema"
+
+export interface TappedTabBarArgs {
+  badge: boolean
+  tab: string
```

Comment:
This should use the `Tab` type you defined elsewhere

```suggestion
  tab: Tab
```